app/models/invenio_rdm_record_converter.rb

```diff
@@ -215,8 +215,8 @@ def resource_type(digitalhub_subtype)
   end
 
   def build_creator_contributor_json(creator)
-    json = @@person_or_org_data[creator]
-    return json if json
+    creatibutor_json = @@person_or_org_data[creator]
```

Comment:
Just to make it more clear exactly what json is being referenced without the need to think about scope as much.